.travis.yml

```diff
@@ -59,13 +59,9 @@ jobs:
   fast_finish: true
   # These need to be exact matches, including whitespace!
   allow_failures:
-    # PHP unit tests (7.3, WordPress trunk)
-    - env: WP_VERSION=trunk  DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1
-    # PHP and JavaScript unit tests (7.3, WordPress trunk, with code coverage)
-    - env: WP_VERSION=latest DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1 RUN_PHPUNIT_COVERAGE=1
     # PHP unit tests (8.0, WordPress trunk)
     - env: WP_VERSION=trunk  DEV_LIB_ONLY=phpunit                               INSTALL_PWA_PLUGIN=1
-      php: nightly
+      php: "nightly"
```

Comment:
Should be 8.0?

Reply:
I'm not sure to be honest, but I'll give it a try. According to this [thread](https://travis-ci.community/t/php-8-0-missing/10132/15) `8.0snapshot` is available, but there is no definitive proof of `8.0` being available as yet.

Reply:
It seems the 8.0 image is now available.